docs/projects-to-work-on.md

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 1
+---
+
+# Projects to Work on
+
+In this section, some projects, issues, improvements or ideas are presented for
+the larger Ada community to work together and make Ada better. However, if you
+are a newcomer, this section is also for you! If you are looking for practical
+topics to train your skill and help the community, this is a great starting
+point.
+
+## How is this list structured?
+
+The proposals are ordered by topic. Within each topic, they are ordered in terms
+of difficulty, that way, regardless of your experience with Ada or with the
+topic at hand, there should always be something for everybody.
+
+Notice that the selection of projects here is not exhaustive and will never
+be. The criteria for adding new points to this list is that the projects or
+topics should have a noticeable impact on the community. Individual ideas or
+personal projects will not be promoted here, but feel free to work on those too!
+
+# List of projects
+
+## Ada Advocacy
+
+Ada does not have a lot going in the marketing department, lets fix that! All
+these proposals apply regardless of your Ada experience, each person will have
+their own take and point of view.
+
+- **Helping newcomers:** Ada has a constant influx of new and curious people
+  wanting to try the language and learn it. One of the most important parts of
+  the learning experience is the community and how it welcomes and aids
+  _newbees._ The Ada community could use a few more people willing to help and
+  guide new programmers with their first few steps into the language.
+- **Blog posts:** writing blogs is always a nice way to share knowledge,
+  experience, tips and anything in between. Ada could use a little more of
+  that. And don't forget to share the blog posts with the community afterwards!
+- **Forums, social media and language discussions:** there are a lot of general
+  discussions going on in the internet, we should make Ada have a bit more
+  presence there (if it is suitable!).
+  - **Example:** the [Awesome WASM Lang
+    repository](https://github.com/appcypher/awesome-wasm-langs) lists all the
+    languages that support targetting WASM. Ada supports WASM! but it is not on
+    the list. It would be great if someone did a Pull Request.
```

Comment:
Well, I actually did it, so better to find a better example. :smile:
A good example should be fixed ASAP, in any case, so don't know what to suggest :smile: Maybe, a done case and not a pending case.

Reply:
Hehehehe... You just fell into the trap :P I wanted to use a real example that someone would have done to show how easy and simple it is and then strike-through the text x)

Thanks for the work! I will now reference it ^^

Reply:
Oh, I just saw that the PR was created long ago... My evil plan did not work that well... But it still counts ^^